### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by masterSplinter01**
Related to #632

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 This document outlines major changes between releases.
 
+## [Unreleased]
+
+### Added
+- Exposure of pool metrics (#615)
+### Changed
+- Improved wallet configuration via `.yaml` config and environment variables (#607)
+### Removed
+### Fixed
```
Comment (KirillovDenis) [inline CHANGELOG.md:12]: I think we can add new section when we need it. Also we should add `Updating from v0.23.0` section, see https://github.com/nspcc-dev/neofs-s3-gw/blob/master/CHANGELOG.md#updating-from-v0220

Comment (masterSplinter01) [reply CHANGELOG.md:12]: I suppose adding of the most used sections in advance will make further appending of new changes a little more friendly. We can remove unused sections when we release.

Comment (masterSplinter01) [reply CHANGELOG.md:12]: Also core dev team fills changelog in the same way: https://github.com/nspcc-dev/neofs-node/blob/master/CHANGELOG.md

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,21 @@
...
+### Updating from v0.23.0
+1. Make sure configuration of wallet is valid. If you use:
+   1. environment variables: set `S3_GW_WALLET_PATH` instead of `S3_GW_WALLET`, 
+   set `S3_GW_WALLET_ADDRESS` instead of `S3_GW_ADDRESS`
+   2. `.yaml` config: to set wallet path use `wallet.path`,
```
Comment (KirillovDenis) [inline CHANGELOG.md:17]:
```suggestion
   2. `.yaml` config: to set wallet path use `wallet.path` instead of `wallet`, 
```

**Review by KirillovDenis — APPROVED** [review]

**Pull request merged**